src/VASL/counters/ASLTranslate.java

```diff
@@ -159,9 +160,18 @@ protected Command moveTarget(GamePiece target) {
   @Override
   protected void translate(Point p) {
       Board b = getMap().findBoard(p);
-      if (getMap().getMapName().equals("Casualties")){  // need to disable one hex moves by CTRL-numberpad because there is no grid in Casbin; VASSAL error results
+      // test for situations where Key Combos won't work: b is null; Casbin and Tray extensions (?) which don't have grid
+      SquareGrid testgrid = new SquareGrid();
+      if (b == null) {
+          // do nothing
+      }
```

Comment:
You've changed what the function does here. The `getMap().getMapName().equals("Casualties")` case used to run regardless of what `b` is.

Reply:
Yes, that was my express intent. If b is null then nothing is going to work and the program should exit the method.